.github/workflows/build.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: windows-latest
```

Comment:
Why?
I'd rather it build on linux if possible just to prove it can run cross-platform (since I tend to develop on Windows, personally). Is there a reason to change it?

Reply:
It is giving an error in Github action 4.0 not supported! so i changed to windows-latest

Reply:
https://github.com/ardalis/GuardClauses/runs/1007887125?check_suite_focus=true